src/components/DashboardMetricChart/DashboardMetricChart.js

```diff
@@ -197,6 +201,19 @@ const DashboardMetricChart = ({
               intervals={intervals}
             />
           </DesktopOnly>
+
+          <ContextMenu
+            showDownload
+            setOptions={setOptions}
+            {...options}
+            data={data}
+            activeMetrics={activeMetrics}
+            chartRef={chartRef}
+            classses={{
+              settingsBtn: styles.settingsBtn
+            }}
```

Comment:
`styles` is the object, so it can be passed directly to `classses` -> `classes={styles}`

Reply:
Correct. But I did not check usage of 'classes' field. Done as made creator of 'ContextMenu'

But will investigate in future